internal/printer/printer.go

```diff
@@ -313,5 +314,28 @@ func (p *printer) printComponentMetadata(doc *astro.Node, source []byte) {
 			p.print(fmt.Sprintf("{ type: 'inline', value: '%s' }", escapeSingleQuote(node.FirstChild.Data)))
 		}
 	}
+	p.print("]")
+
+	// Add Resources (<link> tags)
+	p.print(", resources: [")
```

Comment:
I went with `resources` because the spec refers to the `<link>` tag as “external resources”. How do people feel about this?

Is `externalResources` better? `linkTags`? Open to opinions.

Reply:
`resources` makes sense to me!